rules/dead-code/src/Rector/FunctionLike/RemoveDuplicatedIfReturnRector.php

```diff
@@ -104,12 +104,12 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        foreach ($ifWithOnlyReturnsByHash as $stmts) {
+        foreach ($ifWithOnlyReturnsByHash as $ifWithOnlyReturnByHash) {
```

Comment:
This is a special naming "x by y", e.g. methosByClass.
The result of this single name should be left part infront of by, e.g. methods

Reply:
implemented https://github.com/rectorphp/rector/pull/5696/commits/9a4aa796e13d7bd4dc822440dc784b8d334c04db

Reply:
regenerated https://github.com/rectorphp/rector/pull/5696/commits/594db778483b9e4e090a37333e0efbf737654fcd